tests/components/roku/test_media_player.py

```diff
@@ -56,76 +58,64 @@
 TV_SERIAL = "YN00H5555555"
 
 
-async def test_setup(hass: HomeAssistantType, requests_mock: Mocker) -> None:
+async def test_setup(
+    hass: HomeAssistantType, aioclient_mock: AiohttpClientMocker
+) -> None:
     """Test setup with basic config."""
-    await setup_integration(hass, requests_mock)
+    await setup_integration(hass, aioclient_mock)
 
     entity_registry = await hass.helpers.entity_registry.async_get_registry()
-
     main = entity_registry.async_get(MAIN_ENTITY_ID)
+
     assert hass.states.get(MAIN_ENTITY_ID)
+    assert main
     assert main.unique_id == UPNP_SERIAL
 
 
-async def test_idle_setup(hass: HomeAssistantType, requests_mock: Mocker) -> None:
+async def test_idle_setup(
+    hass: HomeAssistantType, aioclient_mock: AiohttpClientMocker
+) -> None:
     """Test setup with idle device."""
-    with patch(
-        "homeassistant.components.roku.Roku.power_state",
-        new_callable=PropertyMock(return_value="Off"),
-    ):
-        await setup_integration(hass, requests_mock)
+    await setup_integration(hass, aioclient_mock, power=False)
 
     state = hass.states.get(MAIN_ENTITY_ID)
     assert state.state == STATE_STANDBY
 
 
-async def test_tv_setup(hass: HomeAssistantType, requests_mock: Mocker) -> None:
+async def test_tv_setup(
+    hass: HomeAssistantType, aioclient_mock: AiohttpClientMocker
+) -> None:
     """Test Roku TV setup."""
     await setup_integration(
         hass,
-        requests_mock,
+        aioclient_mock,
         device="rokutv",
         app="tvinput-dtv",
         host=TV_HOST,
         unique_id=TV_SERIAL,
     )
 
     entity_registry = await hass.helpers.entity_registry.async_get_registry()
-
     tv = entity_registry.async_get(TV_ENTITY_ID)
+
     assert hass.states.get(TV_ENTITY_ID)
+    assert tv
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5518609)